### PR TITLE
Updates for Version 1.2.1

### PR DESCRIPTION
**Pull request by Daniel-Lam**
Rewrote section on adding `java-logging` module as a dependency, to split it into its two constituent components and list each dependency separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ output.
 
 Detailed documentation can be found [here](docs/logging.md)
 
+## Version
+
+The latest version at the time of writing is 1.2.1.
```
Comment (timja-kainos) [inline README.md]: remove this

File: README.md [hunk 2]
```diff
@@ -17,24 +21,50 @@ stack. It allows a number of configuration options to customize the logging to y
...
 ```
 
 Gradle:
+```groovy
+compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.2.1'
+```
+
+#### java-logging-httpcomponents
 
+Use for formatting log output from HTTP / HTTPS requests made by back-end applications.
```
Comment (timja-kainos) [inline README.md]: Its for adding request IDs to external requests

File: README.md [hunk 3]
```diff
@@ -17,24 +21,50 @@ stack. It allows a number of configuration options to customize the logging to y
...
+</dependency>
+```
+
+Gradle:
 ```groovy
-compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: '1.0.0'
+compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: '1.2.1'
 ```
 
+**Please note:** You will also need to implement a configuration class that adds interceptors for HTTP requests and responses; see https://git.reform.hmcts.net/common-components/payment-app/blob/master/api/src/main/java/uk/gov/hmcts/payment/api/configuration/RestTemplateConfiguration.java for an example, replacing the bean names appropriately.
```
Comment (timja-kainos) [inline README.md]: Don't put internal links in opensourced projects

File: README.md [hunk 4]
```diff
@@ -17,26 +17,45 @@ stack. It allows a number of configuration options to customize the logging to y
 
 ### Basic usage
 
-You should be able to use the module by simply adding it as your project's dependency.
+The module comprises two components; simply add a component as your project's dependency to use it.
```
Comment (timja-kainos) [inline README.md]: there's 3 though

File: README.md [hunk 5]
```diff
@@ -17,26 +17,45 @@ stack. It allows a number of configuration options to customize the logging to y
...
 
-Maven:
+#### java-logging-spring
+
+Use for formatting log output in Spring Boot applications.
```
Comment (timja-kainos) [inline README.md]: it does more than that

File: README.md (hunk 4 above)
Comment (Daniel-Lam) [reply README.md]: There are - unless we remove the (apparently redundant) `dropwizard` component.

**Pull request merged**